### DISCUSSION TITLE
Error  Cannot find module 'webpack'

### DISCUSSION THREAD

**Discussion by amanbabbar-kipl** [webpack/webpack#18643] [Q&A]
I have a Nestjs application when I run Nest build command it fails with the error: Cannot find module 'webpack'

Ubuntu: 22.04.2 LTS x86_64
Node v20.11.0 
Npm v10.2.4

I have to install the webpack package and then restart my app manually.

**Error:**

Error  Cannot find module 'webpack'
Require stack:
- /var/node/nestjs-demo/node_modules/fork-ts-checker-webpack-plugin/lib/issue/issue-webpack-error.js
- /var/node/nestjs-demo/node_modules/fork-ts-checker-webpack-plugin/lib/hooks/tap-after-compile-to-get-issues.js
- /var/node/nestjs-demo/node_modules/fork-ts-checker-webpack-plugin/lib/hooks/tap-start-to-run-workers.js
- /var/node/nestjs-demo/node_modules/fork-ts-checker-webpack-plugin/lib/plugin.js
- /var/node/nestjs-demo/node_modules/fork-ts-checker-webpack-plugin/lib/index.js
- /var/node/nestjs-demo/node_modules/@nestjs/cli/lib/compiler/defaults/webpack-defaults.js
- /var/node/nestjs-demo/node_modules/@nestjs/cli/lib/compiler/webpack-compiler.js
- /var/node/nestjs-demo/node_modules/@nestjs/cli/actions/build.action.js
- /var/node/nestjs-demo/node_modules/@nestjs/cli/actions/index.js
- /var/node/nestjs-demo/node_modules/@nestjs/cli/commands/command.loader.js
- /var/node/nestjs-demo/node_modules/@nestjs/cli/commands/index.js
- /var/node/nestjs-demo/node_modules/@nestjs/cli/bin/nest.js

**Comment by alexander-akait** [comment]
Please run `npm ls webpack`

**Reply by amanbabbar-kipl** [reply]
@alexander-akait 

I got this response when I ran the `npm ls webpack` command. why I have to install webpack manually when its the nest-cli that install webpack internally

nestjs-demo@0.0.1 /var/node/nestjs-demo
├─┬ @nestjs/cli@10.4.2
│ ├─┬ fork-ts-checker-webpack-plugin@9.0.2
│ │ └── webpack@5.93.0 deduped
│ └── webpack@5.92.1
├─┬ ts-loader@9.5.1
│ └── webpack@5.93.0 deduped
└─┬ webpack@5.93.0
  └─┬ terser-webpack-plugin@5.3.10
    └── webpack@5.93.0 deduped

**Reply by alexander-akait** [reply]
Please try to reinstall your deps, if it will not help - open an issue in `@nestjs` repo, we can't fix it here

**Reply by amanbabbar-kipl** [reply]
I deleted **node_modules** and **package-lock.json** and then reinstalled deps multiple times and got the same issue. I have to install webpack package manually. I will open the issue in @nestjs repo as you suggested
